apps/web/pages/my-account/my-orders.vue

```diff
@@ -27,37 +27,44 @@
-            <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.orderId') }}</p>
-            <span class="block typography-text-sm mb-2">{{ orderGetters.getId(order) }}</span>
-          </li>
-          <li>
-            <p class="block typography-text-sm font-medium">
-              {{ $t('account.ordersAndReturns.orderDate') }}
-            </p>
-            <span class="block typography-text-sm mb-2">{{ orderGetters.getDate(order) }}</span>
-          </li>
-          <li>
-            <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.amount') }}</p>
-            <span class="block typography-text-sm mb-2">{{ $n(orderGetters.getPrice(order), 'currency') }}</span>
-          </li>
-          <li v-if="orderGetters.getShippingDate(order)">
-            <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.shippingDate') }}</p>
-            <span class="block typography-text-sm mb-2">{{ orderGetters.getShippingDate(order) }}</span>
-          </li>
-          <li class="flex flex-wrap items-center mb-2">
-            <p class="block typography-text-sm -mb-1.5 font-medium flex-[100%]">
-              {{ $t('account.ordersAndReturns.status') }}
-            </p>
-            <span class="block typography-text-sm flex-1">{{ orderGetters.getStatus(order) }}</span>
-            <SfButton :tag="NuxtLink" size="sm" variant="tertiary" :to="localePath(generateOrderDetailsLink(order))">
-              {{ $t('account.ordersAndReturns.details') }}
-            </SfButton>
-          </li>
-          <UiDivider class="col-span-3 -mx-4 !w-auto md:mx-0" />
-        </ul>
-        <table class="hidden md:block text-left typography-text-sm mx-4">
+
+        <template v-if="!isTablet">
+          <ul class="my-4 last-of-type:mb-0" v-for="(order, index) in data.data.entries" :key="index">
+            <li>
+              <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.orderId') }}</p>
+              <span class="block typography-text-sm mb-2">{{ orderGetters.getId(order) }}</span>
+            </li>
+            <li>
+              <p class="block typography-text-sm font-medium">
+                {{ $t('account.ordersAndReturns.orderDate') }}
+              </p>
+              <span class="block typography-text-sm mb-2">{{ orderGetters.getDate(order) }}</span>
+            </li>
+            <li>
+              <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.amount') }}</p>
+              <span class="block typography-text-sm mb-2">{{ $n(orderGetters.getPrice(order), 'currency') }}</span>
+            </li>
+            <li v-if="orderGetters.getShippingDate(order)">
+              <p class="block typography-text-sm font-medium">{{ $t('account.ordersAndReturns.shippingDate') }}</p>
+              <span class="block typography-text-sm mb-2">{{ orderGetters.getShippingDate(order) }}</span>
+            </li>
```

Comment:
**thought (non-blocking)**: Could use a `v-for` here, so that we don't have to repeat the Tailwind styling every time.

Reply:
same scenario as the above related classes, safe to keep imo